extensions/SemanticForms/includes/SF_ParserFunctions.php

```diff
@@ -601,7 +601,7 @@ static function renderAutoEdit( &$parser ) {
 		return $parser->insertStripItem( $output, $parser->mStripState );
 	}
 
-	static function createFormLink( &$parser, $params, $parserFunctionName ) {
+	static function createFormLink( $parser, $params, $parserFunctionName ) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') The Cyclomatic Complexity of this function "createFormLink" is 32 which is greater than 20 authorized. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1541)
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This function "createFormLink" has 163 lines, which is greater than the 150 lines authorized. Split it into smaller functions. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS138)